src/core/server/saved_objects/import/collect_saved_objects.test.ts

```diff
@@ -38,34 +39,9 @@ Object {
 
   test('collects objects from stream', async () => {
     const readStream = new Readable({
+      objectMode: true,
       read() {
-        this.push('{"foo":true,"type":"a"}');
-        this.push(null);
-      },
-    });
-    const result = await collectSavedObjects({
-      readStream,
-      objectLimit: 1,
-      supportedTypes: ['a'],
-    });
-    expect(result).toMatchInlineSnapshot(`
-Object {
-  "collectedObjects": Array [
-    Object {
-      "foo": true,
-      "migrationVersion": Object {},
-      "type": "a",
-    },
-  ],
-  "errors": Array [],
-}
-`);
-  });
-
-  test('filters out empty lines', async () => {
```

Comment:
This diff is hard to read. The `filters out empty lines` test was removed from here, but added [here](https://github.com/elastic/kibana/pull/39674/files#diff-d4882f00eaa6097afd8e3087b905879cR54)